packages/chaire-lib-common/src/config/__tests__/Preferences.test.ts

```diff
@@ -44,11 +44,13 @@ test('Test current match attributes', () => {
 
 test('Test initialize PreferencesClass with custom attributes', () => {
     const preferences = new PreferencesClass({ data: { foo: 'bar' }, osrmRouting: { directoryPrefix: 'newPrefix' } });
+    // TODO: Decide if this value should be a config or a preference
```

Comment:
It should definitely be a config not a preference. 
What would be the case for a user to configure this ?

Reply:
I meant it more in the way that osrmRouting.directoryPrefix is still in defaultPreferences, so in the future we should decide wether we move it to server.config like the rest. I added similar comments to the other uses of osrmRouting from Preferences.

Reply:
The question is not if we should or not. We definitely should. If you don't want to migrate it now, the comment should read along the lines of "This value also need to be migrated to a config like we did for ..."